paf-mvp-frontend/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "homepage": "https://github.com/criteo/paf-mvp-demo/paf-mvp-frontend#readme",
   "dependencies": {
+    "@types/googletag": "^2.0.0",
```

Comment:
I may be mistaken, but is there any usage of these types definitions? It doesn't seem to be used in `ad-unit-code.ts`

Reply:
You are right. I finally remove it. I will update the PR. Thank you.

Reply:
Fixed